index.js

```diff
@@ -8,11 +8,15 @@ var redent = require('redent');
 var readPkgUp = require('read-pkg-up');
 var loudRejection = require('loud-rejection');
 var normalizePackageData = require('normalize-package-data');
+var getStdin = require('get-stdin');
+var Promise = require('pinkie-promise');
 
 // get the uncached parent
 delete require.cache[__filename];
 var parentDir = path.dirname(module.parent.filename);
 
+global.Promise = Promise;
```

Comment:
`get-stdin` does not support `0.10` and thus does not come bundled with `pinkie-promise`. So either we drop `0.10` in `meow`, or add `pinkie-promise` to `get-stdin` in order to get rid of this.